components/prize_pool/wikis/rainbowsix/prize_pool_custom.lua

```diff
@@ -0,0 +1,39 @@
+---
+-- @Liquipedia
+-- wiki=rainbowsix
+-- page=Module:PrizePool/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Arguments = require('Module:Arguments')
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+
+local PrizePool = Lua.import('Module:PrizePool', {requireDevIfEnabled = true})
+ -- TODO: Move some Widget modules away from Infobox?
+local WidgetInjector = Lua.import('Module:Infobox/Widget/Injector', {requireDevIfEnabled = true})
+-- local LpdbInjector = Lua.import('Module...', {requireDevIfEnabled = true})
+
+local CustomPrizePool = {}
+local CustomWidgetInjector = Class.new(WidgetInjector)
+-- local CustomLpdbInjector = Class.new(LpdbInjector)
+
+-- Template entry point
+function CustomPrizePool.run(frame)
+	local args = Arguments.getArgs(frame)
+	local prizePool = PrizePool(args)
+	prizePool:readInput(args)
```

Comment:
This sequence does not make sense to me. We've already initialized `PrizePool` with `args`, why call yet another function with it? Can we not have `PrizePool` start it themselves?

Reply:
Allow a custom to add customPrizes (and now customConfig) before those are read.
Args input has been removed.